Orange/widgets/visualize/owsilhouetteplot.py

```diff
@@ -907,8 +911,8 @@ def event(self, event):
         return super().event(event)
 
     def sizeHint(self, which, constraint=QSizeF()):
-        spacing = max(self.__spacing * (self.count() - 1), 0)
-        return QSizeF(300, self.__barsize * self.count() + spacing)
+        spacing = max(self.__spacing, 0)
```

Comment:
Why `max(self.__spacing, 0)`? Can `self.__spacing` be negative? I suspect this code was here because of `(self.count() - 1)`.